docs/sdks/sdk-ref-java.md

```diff
@@ -54,14 +49,46 @@ The binary is published on the Maven repository:
 
 ## Initialization
 
+The initialization step depends on the role, as shown in the following table.
+
+| Role                           | Create Instance of Class | Link to Instructions                              |
+|:-------------------------------| :--- |:--------------------------------------------------|
+| Publisher                      | `PublisherUid2Client` | [Usage for Publishers](#usage-for-publishers)     |
+| Advertisers and Data Providers | `IdentityMapClient` | [Usage for Advertisers and Data Providers](#usage-for-advertisers-and-data-providers)     |
+| DSP                            | `BidstreamClient` | [Usage for DSPs](#usage-for-dsps)                 |
+
+You will need to provide the values necessary for the SDK to authenticate with the EUID service.
+
+| Parameter | Description                                                                                | 
+| :--- |:-------------------------------------------------------------------------------------------|
+| `baseUrl/uid2BaseUrl` | The endpoint for the EUID service. See [Environments](../getting-started/gs-environments). | 
+| `clientApiKey` | The API key. See [EUID Credentials](../getting-started/gs-credentials).                    | 
+| `base64SecretKey` | The client secret. See [EUID Credentials](../getting-started/gs-credentials).              |
+
+(**GWH__JN the below table is from the EUID version. The one above is copied from UID2. The legacy EUID version is so different that I'm including it to double-check what the EUID doc should say.**)
+
 The initialization function configures the parameters necessary for the SDK to authenticate with the EUID service. It also allows you to configure retry intervals in the event of errors.
 
 | Parameter | Description | Recommended Value |
 | :--- | :--- | :--- |
 | `endpoint` | The endpoint for the EUID service. | N/A |
 | `authKey` | The authentication token that belongs to the client. For access to EUID, see [Contact Info](../getting-started/gs-account-setup.md#contact-info). | N/A |
 
-## Interface 
+### Interface (GWH__JN section copied from UID2)
+
+(**GWH__JN not sure what we need out of this section and the next which is legacy for EUID. Will EUID folks use the`SharingClient` class? And that "decrypt method" piece is not in the UID2 doc for Java SDK. I left the "Usage for Sharers" section out, but not sure about these encryption and decryption examples, whether they should be in.**)
```

Comment:
`SharingClient` will be used by EUID folks when Sharing becomes supported

Reply:
Updated, thx.